department-of-veterans-affairs/va.gov-cms
# VACMS 1743 & 1744 save and continue

## Description
See #1743 & #1744 

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/87573551-169f7180-c69b-11ea-9251-fabcdf278271.png)
![image](https://user-images.githubusercontent.com/2404547/87573564-1c955280-c69b-11ea-95aa-21a0b3c95c5d.png)
![image](https://user-images.githubusercontent.com/2404547/87574242-1c498700-c69c-11ea-9938-2e6a01ed3577.png)

## QA steps
- As an admin, go to `node/add/full_width_banner_alert` and visually verify that only `Save` button (in form actions section) is present.
- As an admin, go to `node/add/health_care_region_detail_page` and visually verify that save and continue button now reads `Save draft and continue editing`
- As an admin, go to `/node/5676/edit`
- Add a revision log message, set the state to published, and click the Save and continue editing button
- Go to `/node/5676/revisions` and visually verify that the message you entered appears next to the revision you just created and published state is indicated for the revision.
